klient/conf/config.go

```diff
@@ -33,14 +34,33 @@ var DefaultClusterContext = ""
 // New returns Kubernetes configuration value of type *rest.Config.
 // filename is kubeconfig file
 func New(fileName string) (*rest.Config, error) {
-	// if filename is not provided assume in-cluster-config
+	var resolvedKubeConfigFile string
+	context := ResolveClusterContext()
```

Comment:
Please do not shadow the context here with the default package name. Use `kubeContext` instead

Reply:
Ops you are right, sorry. Fixed 🤦